systemd/systemd.go

```diff
@@ -100,8 +100,8 @@ var systemctlCmd = func(args ...string) ([]byte, error) {
 	// output mode, see LP #1885597
 	bs, err := exec.Command("systemctl", args...).CombinedOutput()
 	if err != nil {
-		exitCode, _ := osutil.ExitCode(err)
-		return nil, &Error{cmd: args, exitCode: exitCode, msg: bs}
+		exitCode, exitCodeErr := osutil.ExitCode(err)
```

Comment:
exitCodeErr is set when there's no exit code, so I think runErr (how its called inside ExitCode) is a better name for the var and the field